src/file-1.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:33003/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:33003/project/issues?id=sample&pullRequest=5730&issues=AWdfl0qgZRnp8g6z54J1&open=AWdfl0qgZRnp8g6z54J1)

[ISSUE_KEY]: AWdfl0qgZRnp8g6z54J1